service-worker/windowclient-navigate/index.html

```diff
@@ -0,0 +1,31 @@
+---
+feature_name: "Service Worker: WindowClient.navigate()"
+chrome_version: 49
+feature_id: 5314750808326144
+---
+
+<h3>Background</h3>
+<p>
+  <code>WindowClient.navigate()</code> allows a service worker to cause a web
+  page to navigate to a specific URL.
+</p>
+
+<p>
+  This sample shows how <code>WindowClient.navigate()</code> can be called
+  within a service worker's <code>activate</code> handler, to navigate to
+  <code>activated.html</code>. If your browser supports service workers, and
+  supports <code>WindowClient.navigate()</code>, then you'll be automatically
+  navigated away from this page once the service worker activates.
```

Comment:
For `client.navigate(destination)`, I believe the restriction regarding scope applies to `client`. It needs to be a `WindowClient` that's in scope for the service worker in question. I don't believe that restriction applies to `destination`.

That matches what I'm seeing in practice against M49. It also matches my interpretation of the negative example at https://github.com/slightlyoff/ServiceWorker/issues/752#issue-107657168